spec/models/presenter/auction_spec.rb

```diff
@@ -5,6 +5,136 @@
   let(:ar_bids_by_amount) { ar_auction.bids.order('amount ASC, created_at ASC') }
   let(:auction) { Presenter::Auction.new(ar_auction) }
   let(:user) { FactoryGirl.create(:user) }
+  
+  describe 'internal bid methods' do
```

Comment:
We don't have to do this now, but for the future: it's easier for me to find / read unit tests if they are grouped by the method they are testing. eg:

```ruby

describe '#lowest_bid' do
  context 'when there are no bids' do
    # tests for that method
  end

  context 'when there are multiple bids for the same amount' do
    # tests for that method
  end
end
```

etc etc